### ISSUE TITLE
false positive INFO when directly iterating tuple

### ISSUE THREAD

**Issue by pao** [tonyhffong/Lint.jl#29]
``` Julia
for i in (1, 2, 3)
    @show i
end
```

```
PS C:\Users\patrick\Documents> ~/AppData/Local/julia-0.3.0/bin/julia.exe -e 'using Lint; lintfile(ARGS[1])' lint-test.jl
        lint-test.jl [                    ]    0 INFO   Iteration generates tuples of (Int64,Int64,Int64). N of variables used: 1
```
